mobx/lib/mobx.dart

```diff
@@ -30,7 +30,7 @@ library mobx;
 
 export 'package:mobx/src/api/action.dart';
 export 'package:mobx/src/api/annotations.dart'
-    show action, computed, observable, store;
+    show action, computed, observable, toString;
```

Comment:
I don't see how I can create a storeConfig(toString: false) constant.
I just used @MakeStoreConfig(makeToString: true) with default to false in order to prevent bad surprises for users.

Reply:
toString is an override so I named the parameter makeToString.